src/Tizen.Multimedia.Vision/MediaVision/PoseLandmarkDetector.cs

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the License);
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an AS IS BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System;
+using System.Collections;
+using System.Linq;
+using System.Runtime.InteropServices;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using InteropInference = Interop.MediaVision.Inference;
+using Unmanaged = Interop.MediaVision;
+
+namespace Tizen.Multimedia.Vision
+{
+
+    /// <summary>
+    /// Represents a Landmark in pose detection.
+    /// </summary>
+    /// <since_tizen> 9 </since_tizen>
+    public struct Landmark
+    {
+        /// <summary>
+        /// Represents a location in the 2D space.
+        /// </summary>
+        /// <since_tizen> 9 </since_tizen>
+        public Point location
+        {
+            get;
+            set;
+        }
+        /// <summary>
+        /// Confidence score of point.
+        /// </summary>
+        /// <since_tizen> 9 </since_tizen>
+        public float score
```

Comment:
Please use the PascalCase for the property
```suggestion
        public float Score
```